passes/techmap/flatten.cc

```diff
@@ -275,6 +295,14 @@ struct FlattenPass : public Pass {
 		log("    -wb\n");
 		log("        Ignore the 'whitebox' attribute on cell implementations.\n");
 		log("\n");
+		log("    -noscopeinfo\n");
+		log("        Do not create '$scopeinfo' cells that preserve attributes of cells and\n");
+		log("        modules that were removed during flattening. With this option the\n");
+		log("        'src' attributes is handled like it was before '$scopeinfo' was\n");
+		log("        introduced, i.e. multiple 'src' locations are combined separted by\n");
+		log("        whitespace. Without this option these 'src' locations can be found\n");
+		log("        via the 'cell_src' and 'module_src' attribute of '$scopeinfo' cells.\n");
```

Comment:
I would suggest not referring to how this was handled in the past, just explain source locations would be collected from different points on the hierarchy and combined without distinction on the cell post flattening.

Reply:
(Also FYI strpool uses `|` not whitespace as separator).

Reply:
Right, I must have been thinking of `hdlname`, I'll update the description.